### PR TITLE
The metrics docs

### PR DESCRIPTION
**Pull request by imnotashrimp**
# What changed

Added documentation for logz.io infrastructure monitoring

----

## Pages to review

- https://deploy-preview-345--logz-docs.netlify.com/user-guide/infrastructure-monitoring/
- https://deploy-preview-345--logz-docs.netlify.com/user-guide/infrastructure-monitoring/configure-grafana-drilldown-links.html
- https://deploy-preview-345--logz-docs.netlify.com/user-guide/accounts/manage-the-infrastructure-monitoring-account.html

## Remaining work

- [x] Technical review
- [x] Copy Review

## Post launch

**Do not remove** - To be completed by the docs team upon merge:

no tasks

### CODE REVIEW

File: _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+In the datasource list, choose the Elasticsearch datasource.
+This is the name of your Infrastructure Monitoring account,
+which you set in the [_Manage accounts_](https://app.logz.io/#/dashboard/settings/manage-accounts) page.
+(By default, it's the name of your main account, followed by "_metrics".)
+
+For help configuring the query,
+see [Query Editor](https://grafana.com/docs/grafana/latest/guides/basic_concepts/#query-editor)
+from Grafana.
+{:.info-box.read}
+
```
Comment (daniel-tk) [inline _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md]: I don't see this row here, but there might be a redundant "is" after "$cluster"?
image

File: _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+Click the _Visualization_ icon to the left to see visualization options.
+
+![Panel configuration, visualization  options]({{site.baseurl}}/images/grafana/panel-config--query--visualization.png)
+
+Configure the visualization as you normally would.
+
+##### Configure the drilldown
+
```
Comment (daniel-tk) [inline _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md]: Sorry I don't see this row here either (I guess I'm not familiar enough with github) but this row is inaccurate:

image
It's very confusing, but the query in the panel is not related to the query in the drilldownlink. You have to write your query again in the Url params box as" query=....", and the two queries might differ from each other.
So you have one query for the panel and then you make another one for the drilldownlink.

File: _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+Save your dashboard.
+(The _Save dashboard_ button is in the toolbar in the upper right corner.)
+
+![Drilldown link]({{site.baseurl}}/images/grafana/panel-drilldown-link.png)
+
+Test your new drilldown link
+by hovering over the upper left corner of the panel,
+and then clicking **Explore in Kibana**.
+
+</div>
```
Comment (daniel-tk) [inline _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md]: A great page!! You truly are a really quick learner :)

File: _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+One of the most useful features you'll come across
+in Logz.io Infrastructure Monitoring
+is drilldown links.
+With drilldown links,
+if you see something funky,
+you can click a link
+and go straight to the related logs in Kibana.
+
+Drilldown links are preconfigured on any dashboards managed by Logz.io.
```
[the remark this answers is not in the document]
Comment (imnotashrimp) [reply _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md]: Fixed, Will be in next push

File: _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md (hunk 1 above)
Comment (imnotashrimp) [reply _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md]: Fixed

File: _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md (hunk 2 above)
Comment (imnotashrimp) [reply _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md]: Thanks for the clarification! Fixed in the next push

File: _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md (hunk 3 above)
Comment (imnotashrimp) [reply _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md]: 🤓

File: _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+If you're setting up your own dashboard,
+you'll need to configure drilldown links.
+This doc shows you how.
+
+#### To configure Grafana drilldown links
```
[the remark this answers is not in the document]
Comment (imnotashrimp) [reply _source/user-guide/metrics-analytics/configure-grafana-drilldowns.md]: Addressed this under the "to configure variables" headings.

**Pull request merged**